spec/controllers/reports_controller_spec.rb

```diff
@@ -48,7 +34,11 @@
     end
     it "allows downloading a CSV file" do
       get :ephemera_data, params: { project_id: project.id }, format: "csv"
-      expect(response.body).to eq(data)
+      # expect(response.body).to eq(data)
```

Comment:
I think this is a good change, but it would be good to cover roughly the same amount of detail above to make sure that the whole set of fields is indeed being exported. And removing the commented-out line above would be good too.